dml_events.go

```diff
@@ -15,17 +15,22 @@ import (
 
 type RowData []interface{}
 
-// The mysql driver never actually gives you a uint64 from Scan, instead you
-// get an int64 for values that fit in int64 or a byte slice decimal string
-// with the uint64 value in it.
+// The old version mysql driver never actually gives you a uint64 from Scan,
```

Comment:
Why the old version? Did something change in the new version? When?

Reply:
A:
* int unsigned and bigint unsigned both
* rechecking and trying to reproduce this

Reply:
updated.
It's go-mysql which gives right uint64.